src/element/textElement.ts

```diff
@@ -221,44 +233,51 @@ export const handleBindTextResize = (
       baseline: nextBaseLine,
     });
     if (!isArrowElement(container)) {
-      updateBoundTextPosition(
+      const { x, y } = computeBoundTextPosition(
         container,
         textElement as ExcalidrawTextElementWithContainer,
       );
+      mutateElement(textElement, { x, y });
```

Comment:
I'd get in the habit of not destructuring + initializing variables + constructing new objects, if not necessary. Here, you can just pass the `computeBoundTextPosition()` as 2nd arg to `mtuateElement` directly.

Not critical, but still somewhat a hot path.